### ISSUE TITLE
Cannot destroy created gates

### ISSUE THREAD

**Issue by Ayasu** [Raphfrk/ServerPort#17]
Trying to destroy gates, and it will not let me, even with my name in the admin list file.

**Comment by Raphfrk** [comment]
Is the gate in the spawn protection area?

Also, can you break the blocks or do they regenerate?

**Comment by Ayasu** [comment]
It's not in a protected area. I can break the signs, but when I break the blocks, they instantly regenerate.

**Comment by Raphfrk** [comment]
Not sure what is happening.

I tested it yesterday using the latest version of the plugin and also 670 of CraftBukkit and it seemed to work.

The admin_list file shouldn't be case sensitive, but it might be worth making sure that the name matches exactly.

Also, it is one name per line in the file.

**Comment by Ayasu** [comment]
My name, at least, is exactly the same as it appears when I log in. I mean, is there something I'm doing wrong to destroy a gate? Breaking the blocks should work just fine, correct?

**Comment by Raphfrk** [comment]
Yeah, breaking one of the blocks which make up the gate should break it.

Are you using the latest version of CraftBukkit?

**Comment by Ayasu** [comment]
I should be. I'm running my server off of a servercraft.co server. I
just switched to them, and they seem like they keep everything up to
date pretty well. (I clicked the update to latest version of
CraftBukkit button just the other night)

On Mon, Apr 11, 2011 at 9:57 AM, Raphfrk
reply@reply.github.com
wrote:

> Yeah, breaking one of the blocks which make up the gate should break it.
> 
> Are you using the latest version of CraftBukkit?
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Raphfrk/ServerPort/issues/17#comment_984083

## 

~BJ Mansfield~

**Comment by Raphfrk** [comment]
I will have a look again this evening.

**Comment by Raphfrk** [comment]
Hmm, still works for me.  I deleted all the files in the permissions directory and just left admin_list.txt with one line in the file.

Are you use you are using the right file/file location

It needs to be:

Directory: <server-directory>/plugins/ServerPort/permissions
Filename: admin_list.txt

**Comment by Ayasu** [comment]
I just deleted the plugin entirely and I'm gonna reinstall it. The
plugin was installed correctly -- the server admin tool does that
automatically. You said I only need the admin_list.txt? Well, I'm also
having an issue where people can't actually use the portals unless I
make them an admin, as well.

Am I doing something wrong?

On Mon, Apr 11, 2011 at 2:44 PM, Raphfrk
reply@reply.github.com
wrote:

> Hmm, still works for me.  I deleted all the files in the permissions directory and just left admin_list.txt with one line in the file.
> 
> Are you use you are using the right file/file location
> 
> It needs to be:
> 
> Directory: <server-directory>/plugins/ServerPort/permissions
> Filename: admin_list.txt
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Raphfrk/ServerPort/issues/17#comment_985625

## 

~BJ Mansfield~

**Comment by Ayasu** [comment]
Okay, I deleted all the files in the directory for the mod, deleted
the mod and re-installed it. But it still somehow "remembers" a gate
that doesn't exist anymore. I'm getting this error message ---
 "creepertop_" already in use". When I use the other linked portal,
creeperbottom, it puts the water for the portal in a place where the
portal shouldn't be.

On Mon, Apr 11, 2011 at 7:02 PM, BJ Mansfield iamtheblackmage@gmail.com wrote:

> I just deleted the plugin entirely and I'm gonna reinstall it. The
> plugin was installed correctly -- the server admin tool does that
> automatically. You said I only need the admin_list.txt? Well, I'm also
> having an issue where people can't actually use the portals unless I
> make them an admin, as well.
> 
> Am I doing something wrong?
> 
> On Mon, Apr 11, 2011 at 2:44 PM, Raphfrk
> reply@reply.github.com
> wrote:
> 
> > Hmm, still works for me.  I deleted all the files in the permissions directory and just left admin_list.txt with one line in the file.
> > 
> > Are you use you are using the right file/file location
> > 
> > It needs to be:
> > 
> > Directory: <server-directory>/plugins/ServerPort/permissions
> > Filename: admin_list.txt
> > 
> > ## 
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/Raphfrk/ServerPort/issues/17#comment_985625
> 
> ## 
> 
> ~BJ Mansfield~

## 

~BJ Mansfield~

**Comment by Raphfrk** [comment]
The gates files are stored in the world directory.

/world/serverport/gates

To allow everyone to use the gates, there should be a file in the permissions directory, with a name 

use_gate_type_list.txt

This has a list of permissions.  If you add one line with 

```
*.*.*.*
```

will give everyone permission with all gates.

Btw, the admin file is actually called: admins_list.txt

Also, there is a destroy_gate_list.txt file which allows gates to be destroyed (but admins should still be able to do it).
